src/Resources/AuthenticationLogResource.php

```diff
@@ -88,8 +89,14 @@ public static function table(Table $table): Table
                         if (! $record->authenticatable_id) {
                             return new HtmlString('&mdash;');
                         }
+                        $authenticableEditRoute = '#';
+                        if (Route::has('filament.'.FilamentAuthenticationLogPlugin::get()->getPanelName().'.resources.'.Str::plural((Str::lower(class_basename($record->authenticatable::class)))).'.edit', ['record' => $record->authenticatable_id])) {
+                            $authenticableEditRoute = route('filament.'.FilamentAuthenticationLogPlugin::get()->getPanelName().'.resources.'.Str::plural((Str::lower(class_basename($record->authenticatable::class)))).'.edit', ['record' => $record->authenticatable_id]);
+                        }else if (config('filament-authentication-log.user-resource')) {
+                            $authenticableEditRoute = config('filament-authentication-log.user-resource')::getUrl('edit',['record' =>$record->authenticatable_id]);
+                        }
```

Comment:
Hi @KamranBiglari

Thank you for this contribution! :)

I think the `Route::has()` uses the route name.
Would you mind updating it like this:

```php
$routeName = 'filament.'.FilamentAuthenticationLogPlugin::get()->getPanelName().'.resources.'.Str::plural((Str::lower(class_basename($record->authenticatable::class)))).'.edit';

if (Route::has($routeName)) {
    $authenticableEditRoute = route($routeName, ['record' => $record->authenticatable_id]);
} else if (config('filament-authentication-log.user-resource')) {
    $authenticableEditRoute = config('filament-authentication-log.user-resource')::getUrl('edit',['record' =>$record->authenticatable_id]);
}
```